py/examples/multi_client_geoimage/config.json

```
@@ -0,0 +1,33 @@
+{
+  "configs": [
+    {
+      "name": "gps",
+      "port": 3001,
+      "host": "localhost"
+    },
+    {
+      "name": "oak0",
+      "port": 50010,
+      "host": "localhost"
+    },
+    {
+      "name": "multi_subscriber",
+      "subscriptions": [
+        {
+          "uri": {
+            "path": "/rgb",
```

Suggest to use a lower resolution image stream as the default (in general). Will reduce issues induced by running this on a remote PC
```suggestion
            "path": "/left",
```